frontend/src/components/project/project.jsx

```diff
@@ -11,6 +11,11 @@ let Project = React.createClass({
     };
   },
 
+  componentDidMount() {
+    let id = this.props.params.id;
```

Comment:
the project id is available here - we would signal an action here that would fetch project details for this project id.